src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -281,7 +281,11 @@ private Pod getPodTemplate(KubernetesSlave slave, Label label) {
         {
             String[] parts = selector.split("=");
 
-            builder = builder.put(parts[0], parts[1]);
+            if (parts.length == 2) {
+                if (parts.length == 2) {
```

Comment:
Ups, don't know how that happened. Thanks.
El 18/3/2016 15:19, "Oleg Nenashev" <notifications@github.com> escribió:

> In
> src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java
> <https://github.com/jenkinsci/kubernetes-plugin/pull/55#discussion_r56662197>
> :
>
> > @@ -281,7 +281,11 @@ private Pod getPodTemplate(KubernetesSlave slave, Label label) {
> >          {
> >              String[] parts = selector.split("=");
> >
> > -            builder = builder.put(parts[0], parts[1]);
> > +            if (parts.length == 2) {
> > +                if (parts.length == 2) {
>
> [image: :ant:] double check of a single field.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/kubernetes-plugin/pull/55/files/7011b5c24bb2c360ec7a1400a298709c98d335ea#r56662197>
>